### PR TITLE
Bug 1193940 - Prevent mode switch from changing while camera is configuring.

### PR DESCRIPTION
**Pull request by aosmond**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aac3171ad0a3dad4d1f0bb08ba2ef9fa2151478b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=43ac40e9757dba037705afbd309f9581759b39f4

File: apps/camera/js/views/controls.js [hunk 1]
```diff
@@ -121,8 +136,25 @@ module.exports = View.extend({
       recording ? 'stop-capture-button' : 'capture-button');
   },
 
-  onSwitchSnapped: function(edges) {
-    var mode = this.switchPositions[edges.x];
+  onSwitchEnded: function() {
+    var delta;
+    var now = new Date().getTime();
```
Comment (justindarc) [inline apps/camera/js/views/controls.js]: `Date.now()`

File: apps/camera/js/views/controls.js [hunk 2]
```diff
@@ -131,15 +163,13 @@ module.exports = View.extend({
...
-  onSwitchTapped: function(e) {
-    e.preventDefault();
-    e.stopPropagation();
-    debug('switch tapped');
-    this.onSwitchChanged();
-  },
-
   onSwitchTranslate: function(e) {
-    this.setSwitchIcon(e.position.ratio.x);
+    var now = new Date().getTime();
```
Comment (justindarc) [inline apps/camera/js/views/controls.js]: `Date.now()`

**Pull request closed without merging**